RELEASE_NOTES.rst

```diff
@@ -455,8 +510,8 @@ Significant Changes
 Make ``RotatingFilehandler`` used in ``DagProcessor`` non-caching (#27223)
 """"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
-  In case you want to decrease cache memory when ``CONFIG_PROCESSOR_MANAGER_LOGGER=True``, and you have your local settings created before,
-  you can update ``processor_manager_handler`` to use ``airflow.utils.log.non_caching_file_handler.NonCachingRotatingFileHandler`` handler instead of ``logging.RotatingFileHandler``. (#27065)
+In case you want to decrease cache memory when ``CONFIG_PROCESSOR_MANAGER_LOGGER=True``, and you have your local settings created before,
```

Comment:
Why these changes on previous release notes? I think the indentation here is for a release  cc @jedcunningham

Reply:
The indentation impacts the rendering of the releases notes, this is a minor thing that just cleans the release notes up a bit:
[With indentation](https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#make-rotatingfilehandler-used-in-dagprocessor-non-caching-27223)
[No indentation](http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow/latest/release_notes.html#make-rotatingfilehandler-used-in-dagprocessor-non-caching-27223)

Reply:
Yes Jed cleaned the Release note to be more consistant.